Jenkinsfile

```diff
@@ -7,75 +7,58 @@
 // We could do a single "checkout scm" and load the groovy script, but to do
 // that one has to allocate a node first, which would stay busy until the end of
 // the parallel pipeline
+
+void buildOnYoctoNode(String variant, String image, boolean checkSmokeTests = false) {
+    node("Yocto") {
+        boolean smokeTests = false
+
+        if (checkSmokeTests) {
+            smokeTests = env.SMOKE_TEST == "true"
+        }
+
+        checkout scm
+        def manifests = load "ci-scripts/yocto.groovy"
+        manifests.buildManifest(variant, image, smokeTests)
+    }
+}
+
+void buildOnYoctoNodeNightly(String variant, String image, boolean checkSmokeTests = false) {
+    boolean nightlyBuild = env.NIGHTLY_BUILD == "true"
+
+    if (nightlyBuild) {
+        buildOnYoctoNode(variant, image, checkSmokeTests)
```

Comment:
The check of this env var has been removed, so the behavior is no longer as it used to be. This should be changed so it checks the env var as the buildOnYoctoNode function does.

Reply:
Oh never mind it is actually checked since this calls the other function. However you do set it always when running qemu, which is not how it used to be.